solvers/relay_constellation/mclp_teg_contact_plan/src/milp_scheduler.py

```
@@ -0,0 +1,250 @@
+if TYPE_CHECKING:
+    from typing import Iterable
+
+
+def milp_scheduler_available() -> bool:
+    """Return True if PuLP and CBC are available."""
+    try:
+        import pulp
+
+        solver = pulp.PULP_CBC_CMD(msg=False, timeLimit=1)
+        return solver.available()
+    except Exception:
```

Ruff (0.15.11)

[warning] Do not catch blind exception: `Exception`

(BLE001)